### PR TITLE
Feature/280421 header

### PR DESCRIPTION
**Pull request by mvn-cuongnguyen2-dn**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          <div class="card">
+            <img src="" class="card-img-left" alt="...">
+            <div class="card-body">
+              <h5 class="card-title">Card title</h5>
+            </div>
+          </div>
+        </div>
+
+
+
```
Comment (ghost) [inline index.html:70]: Please remove line 68-70

File: index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        <div class="newsfeatured article-list">
+          
+        </div>
+    </section>
+  </div>
+  
+</body>
+
+</html>
```
Comment (ghost) [inline index.html:79]: Please add new line in the end of file

File: index.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            <li class="iteam-menu"><a>Xuất bản</a></li>
+            <li class="iteam-menu"><a>Sách</a></li>
+            <li class="iteam-menu"><a>Tác giả</a></li>
+            <li class="iteam-menu"><a>Xã hội</a></li>
+            <li class="iteam-menu"><a>Thế giới</a></li>
+            <li class="iteam-menu"><a>Kinh doanh</a></li>
+            <li class="iteam-menu"><a>Công nghệ</a></li>
+            <li class="iteam-menu"><a>Thể thao</a></li>
+            <li class="iteam-menu"><a>Giải trí</a></li>
+            <li class="iteam-menu open-news">...</li>
```
Comment (ghost) [inline index.html:35]: item not iteam

File: index.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+          </ul>
+        </nav>
+        <div class="toolbox">
+          <div class="search-wrap">
+            <input type="text" >
```
Comment (ghost) [inline index.html:40]: `<input type="text"/>`

File: scss/layout/_header.scss [hunk 5]
```diff
@@ -0,0 +1,43 @@
+.container-max-widths{
+    max-width: 1200px;
+    margin: auto;
```
Comment (ghost) [inline scss/layout/_header.scss:3]: Class `.container` had this style please use instead of this class

File: scss/layout/_header.scss [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+}
+header{
+    .page-wrap{
+        height: 50;
```
Comment (ghost) [inline scss/layout/_header.scss:7]: miss `px`

File: scss/layout/_header.scss [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+                    }
+                    &:hover{
+                        cursor: pointer;
+                        border-bottom: 1px solid #000 ;
+                    }
+                }
+            }
+        }
+    }
+}
```
Comment (ghost) [inline scss/layout/_header.scss:43]: Please insert a white space before `{` and after name class

File: index.html [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+    <header>
+      <div class="page-wrap">
+        <div class="logo">
+          <a href="#">
+            <img src="./img/logo-zing-home.svg" class="card-img-left" alt="...">
+            <span>knowledge online</span>
+          </a>
+        </div>
+        <nav class="menu-top">
+          <ul>
```
Comment (ghost) [inline index.html:25]: add class="menu-list"

File: index.html (hunk 3 above)
Comment (ghost) [reply index.html:35]: should change to class="menu-item"

File: index.html (hunk 2 above)
Comment (mvn-cuongnguyen2-dn) [reply index.html:79]: don't understand

**Pull request merged**